nav2_bt_navigator/behavior_trees/navigate_w_replanning_and_recovery.xml

```diff
@@ -10,12 +10,18 @@
         <RateController hz="1.0">
           <RecoveryNode number_of_retries="1" name="ComputePathToPose">
             <ComputePathToPose goal="{goal}" path="{path}" planner_id="GridBased"/>
-            <ClearEntireCostmap name="ClearGlobalCostmap-Context" service_name="global_costmap/clear_entirely_global_costmap"/>
+            <ReactiveFallback name="ComputePathToPoseRecoveryFallback">
```

Comment:
I know this was added to more closely mimic the now deleted `navigate_w_replanning_and_round_robin_recovery.xml ` but I am having trouble understanding why we should be wrapping the  `clear_entirely_global_costmap` with this reactive fallback.

The way I understand it is:
1. Run clear global costmap
2. Interrupt this clearing if the goal is updated. 

But, in real life does the clear costmap service take long enough for this pre-emption to matter to the end user?

Reply:
I think its so that we don't bother performing the recovery if we have a new goal